lang/step.md

```diff
@@ -132,7 +132,7 @@ For now, that is just a pointer (but this might have to change).
 Place evaluation ensures that this pointer is always dereferenceable (for the type of the place expression).
 
 ```rust
-type Place<M: Memory> = Pointer<M::Provenance>;
```

Comment:
Rusts type aliases don't support bounds as of now.
We could implement something in specr, but I think this change is fine too.

Reply:
This will probably make the Coq translator unhappy, but yeah I guess it works for now.